src/js/base/module/Buttons.js

```diff
@@ -388,7 +388,13 @@ export default class Buttons {
           checkClassName: this.options.icons.menuCheck,
           items: this.options.fontSizes,
           title: this.lang.font.size,
-          click: this.context.createInvokeHandlerAndUpdateState('editor.fontSize'),
+          template: function (item) {
```

Comment:
Please use an arrow style function to use `this` in below function scope.
e.g. `template: (item) => { ... }`